README.md

```diff
@@ -26,14 +26,20 @@ _For installation instructions and more examples of usage, please refer to the [
 
 ### Currently supported provisioners
 
-| Provisioner | Description |
-| :--- | :--- |
-| eks | provision nodes on EKS |
-| eks-managed | provision managed node groups on EKS |
+| Provisioner | Description | Supported?
+| :--- | :--- | :---: |
+| eks-cf | provision nodes on EKS using cloudformation | ✅ |
+| eks-managed | provision managed node groups for EKS | ✅ |
+| eks-fargate | provision a cluster to run pods on fargate|✅|
```

Comment:
Can you undo this one, and expand the existing table? I removed the "Supported?" column 😆 and modified the provisioner to reflect ekscf deprecation

Reply:
OK. done.